index.html

```diff
@@ -15,8 +15,78 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
+
+  <header></header>
+
+
+  <div class="container">
+
+
+          <div class="row header-div">
+            <div class="column-half">
+             <h3 class="head-box-text">Code Journal</h3>
+            </div>
+          </div>
```

Comment:
This should be inside the `header` element. You will need an additional element with the `container` class to hold the main content of the page. Also, your indentation in this document is incorrect with a lot of empty whitespace, please clean that up too.